integr8ly/installation
# INTLY-971 move to new RH-SSO 7.3.2

## Additional Information
We needed to change the default scopes for launcher GH IDP

## Verification Steps


install on RHPDS using the following command 

Note there is an issue with GH and Code Ready. However Code Ready is being updated so it is currently just a known issue until we can prove the issue still exists. 
```
ansible-playbook -i inventories/hosts.template playbooks/install.yml -e 'ns_prefix=openshift-' -e 'eval_self_signed_certs=true' -e 'eval_seed_users_count=1' -e 'rhsso_operator_release_tag=1.4.0-rc1'
```
Ensure 7.3.2.GA is installed as the backing SSO
You should then link up GH and launcher following the doc. 
Check you can login to each service as an evals user from the webapp
Ensure you can link to your GH account from Launcher
Code Ready will not be able to import GH stuff currently (we are waiting for the update to Code Ready before taking any further action)

## Is an upgrade task required and are there additional steps needed to test this?


There is a separate upgrade task
